revisions-cli.php

```diff
@@ -303,7 +303,11 @@ public function generate( $args = array(), $assoc_args = array() ) {
 			$p = get_post( $post_id );
 			$content = $p->post_content;
 			for ( $i = 0; $i < $count; $i++ ) {
-				$content .= '&nbsp;';
+				if ( substr($content,-6) == '&nbsp;' ) {
```

Comment:
Can you use [yoda conditions](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#yoda-conditions) here, and add some [spaces](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage) per WP's PHP Coding Standards?